### DISCUSSION TITLE
Contributing to NocoDB [Oct 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4214] [Discord Threads]
> 🙋‍♂️ **ahigg**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Some of the details here. Feel free to get in touch if you need help   
####    
#### https://github.com/nocodb/nocodb/blob/develop/.github/CONTRIBUTING.md  
#  
> 👨‍🦲 **dstala**  
#### You can also help us by creating issue in GitHub & assign to yourself if you are already working on it  
#  
> 🙋‍♂️ **ahigg**  
#### Thank you dstala. I have created an issue at https://github.com/nocodb/nocodb/issues/4024  
#  
> 👨‍🦲 **dstala**  
#### <@1018298557310193674> if you will be working on this, i will assign it to you? you have changes prepared already?  
#  
> 🙋‍♂️ **ahigg**  
#### Yes feel free to assign to me. I have changes but still need to create tests and validate.  
#  
> 🙋‍♂️ **wkw**  
#### assigned  
#  
> 🙋‍♂️ **ahigg**  
#### Hi <@857998597474484295>. Looks like someone jumped in on the issue assigned to me with a PR. As I mentioned I am new to contributing - is this normal? Does it happen a lot? Should I have flagged somehow that I was working on it?  
#  
> 🙋‍♂️ **wkw**  
#### i saw there s a PR. is it made by u ?  
#  
> 🙋‍♂️ **wkw**  
#### oh  
#  
> 🙋‍♂️ **wkw**  
#### i was just wondering  
#  
> 🙋‍♂️ **ahigg**  
#### No not made by me  
#  
> 🙋‍♂️ **ahigg**  
#### The PR doesn't have any tests, which is what I am working on and need some assistance with.  
#  
> 🙋‍♂️ **wkw**  
#### i believe you got the same fix  
#  
> 🙋‍♂️ **wkw**  
#### i ll take his PR and can you add the testing part?  
#  
> 🙋‍♂️ **ahigg**  
#### How would we do the testing part?  
#  
> 🙋‍♂️ **ahigg**  
#### Same process? Create an Issue, etc?  
#  
> 🙋‍♂️ **wkw**  
#### you can create a PR and link to the same issue  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1029656039316000809)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
